src/sentry/snuba/sessions.py

```diff
@@ -191,13 +191,21 @@ def get_release_adoption(project_releases, environments=None, now=None):
         filter_keys=filter_keys,
         referrer="sessions.release-adoption-list",
     )["data"]:
-        total = total_users.get(x["project_id"])
-        if not total:
-            adoption = None
-        else:
-            adoption = float(x["users"]) / total * 100
+        # Users Adoption
+        total_users_count = total_adoption.get(x["project_id"])[0]
```

Comment:
This will error if the .get returns None. Should make a none check and handle that case.